### PR TITLE
offlineimap: 6.5.7

### PR DESCRIPTION
**Pull request by MindTooth**
With added conformity.

Side note:  Best way to run a proper test?  Not used to the test portion.

### CODE REVIEW

File: Library/Formula/offline-imap.rb [hunk 1]
```diff
@@ -1,9 +1,8 @@
-require "formula"
-
 class OfflineImap < Formula
   homepage "http://offlineimap.org/"
-  url "https://github.com/OfflineIMAP/offlineimap/archive/v6.5.6.tar.gz"
-  sha1 "cced67060be2922ca3ce056839b2aa1fe6b776a1"
+  desc "OfflineIMAP is to dispose your mailbox(es) as a local Maildir(s)."
```
Comment (DomT4) [inline Library/Formula/offline-imap.rb]: We're not adding descriptions to existing formulae for now. It's going to be done in bulk in the near future.

**Comment by MindTooth** [comment]
@DomT4 Fixed :-)

File: Library/Formula/offline-imap.rb [hunk 2]
```diff
@@ -14,6 +12,10 @@ def install
     bin.install_symlink libexec+"offlineimap.py" => "offlineimap"
   end
 
+  test do
+    system bin/"offlineimap", "--version"
```
Comment (mikemcquaid) [inline Library/Formula/offline-imap.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

Comment (mikemcquaid) [inline Library/Formula/offline-imap.rb]: Also, can you move it below `caveats` regardless? Thanks!

**Comment by MindTooth** [comment]
Going out now, but I'll look later tonight.  But honestly, not sure if I can come up with a better test.  Sorta extend my beyond scope.  But I'll at least try.

**Comment by mikemcquaid** [comment]
@MindTooth Thanks!

**Comment by MindTooth** [comment]
@mikemcquaid:  Really not sure how to create a proper test.  The only way I see, is having a dummy IMAP account, and print out a dummy conf-file to use for testing.  As for now, I guess as long as it runs, it's okay.  But I do see that just pulling the version isn't enough.

Thoughts?

**Comment by bfontaine** [comment]
There are some formulae where we can’t really write a test better than `--version` or `--help` because they rely on external (non-http) services, and I think this is one of them. I just looked at `offlineimap -h` and even `dry-run` needs access to an IMAP account. @mikemcquaid, thoughts?

**Comment by mikemcquaid** [comment]
Cool, that's fine :+1:

**Comment by DomT4** [comment]
Why aren't we placing this: `prefix.install "offlineimap.conf", "offlineimap.conf.minimal"` in `etc` by the way? Oversight or?

**Comment by MindTooth** [comment]
I'll update it later today.  etc.install correct?

**Comment by mikemcquaid** [comment]
Yup.

**Comment by MindTooth** [comment]
Does it look okay now? :-)

**Comment by DomT4** [comment]
It's good :+1:. Merged in https://github.com/Homebrew/homebrew/commit/eb13fad7eb0f89426ce1e867259ae283a2ed00bd, Thanks for your contribution to Homebrew, we appreciate it! :smiley_cat:.

**Pull request closed without merging**